test/gtest/ucm/malloc_hook.cc

```diff
@@ -771,7 +771,17 @@ class mmap_hooks {
 
         /* 8. sbrk call - single thread only */
         {
-            if (!RUNNING_ON_VALGRIND && m_num_threads < 2) {
+            int num_threads = 0;
+            ucs::enum_threads(enum_threads_cb, &num_threads);
+            // check for real number of threads:
+            // test is actual only in case if 3 threads are in process:
+            // 1. main thread
+            // 2. watchdog thread
+            // 3. test thread
+
+            if (!RUNNING_ON_VALGRIND &&
+                (m_num_threads < 2) &&
```

Comment:
still need to check m_num_threads? or could it be assertion?

Reply:
still need: threads are created one-by-one, and first created thread will detect only 3 threads, continue test to sbrk, and at this moment new threads may be created which may cause error